### PR TITLE
feat(example): list by id and query by id

### PR DESCRIPTION
**Pull request by Jimexist**
**What this PR does**
add query by id and list by id example part

**why we need it**

**Special notes for your reviewer**:

### CODE REVIEW

File: examples/swapi/pkg/api/people/get_by_id.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	return definition.Definition{
+		Method: definition.Get,
+		Function: func(ctx context.Context, id int) (model.Person, error) {
+			index := sort.Search(len(people), func(i int) bool {
+				return people[i].Id > model.Identity(id)
+			})
+			if index < len(people) && people[index].Id == model.Identity(id) {
+				return people[index], nil
+			}
+			return model.Person{}, fmt.Errorf("not found")
```
Comment (walktall) [inline examples/swapi/pkg/api/people/get_by_id.go]: Should use nirvana/errors here.

File: examples/swapi/pkg/api/people/get_by_id.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
+package people
+
+import (
+	"context"
+	"github.com/caicloud/nirvana/examples/swapi/pkg/model"
+	"sort"
+	"github.com/caicloud/nirvana/definition"
```
Comment (walktall) [inline examples/swapi/pkg/api/people/get_by_id.go]: import format shuold obey the go conventions.

Comment (Jimexist) [reply examples/swapi/pkg/api/people/get_by_id.go]: done

File: examples/swapi/pkg/api/people/get_by_id.go (hunk 1 above)
Comment (Jimexist) [reply examples/swapi/pkg/api/people/get_by_id.go]: see https://github.com/caicloud/nirvana/issues/84

**Pull request merged**